be/src/common/status.h

```diff
@@ -5,13 +5,13 @@
 #pragma once
 
 #include <fmt/format.h>
+#include <glog/logging.h>
+
+#include <boost/stacktrace.hpp>
```

Comment:
why boost is behind glog? Does the format not sort includes according to lexical order？

Reply:
Our clang-format style is based on [Google Code Style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) (`BasedOnStyle: Google` in `.clang-format`). The rules doesn't apply lexical order simply.